### PR TITLE
Cross scala 2.13 build

### PR DESCRIPTION
**Pull request by marekzebrowski**

### CODE REVIEW

File: project/Dependencies.scala [hunk 1]
```diff
@@ -13,8 +13,8 @@
 import sbt._
 
 object Dependencies {
-  val catsEffect    = "org.typelevel"      %% "cats-effect"          % "1.3.1"
-  val cats          = "org.typelevel"      %% "cats-core"            % "1.6.1"
+  val catsEffect    = "org.typelevel"      %% "cats-effect"          % "2.0.0"
+  val cats          = "org.typelevel"      %% "cats-core"            % "2.0.0"
```
Comment (chuwy) [inline project/Dependencies.scala]: Any reason to not use 2.1.0?

**Review by chuwy — APPROVED** [review]
That's awesome, thanks @marekzebrowski!

File: project/Dependencies.scala (hunk 1 above)
Comment (marekzebrowski) [reply project/Dependencies.scala]: 2.1.0 is not published for Scala 2.11, so moving forward requires dropping support for that version

Comment (chuwy) [reply project/Dependencies.scala]: I think we should be ready to make this sacrifice. Many downstream libraries (like SCE) already dropped 2.11.

Comment (marekzebrowski) [reply project/Dependencies.scala]: ok, done

**Comment by chuwy** [comment]
Cherry-picked into https://github.com/snowplow-incubator/scala-lru-map/pull/28. Will be released shortly. Thank you very much again!

**Pull request closed without merging**